manual/azure-sru-19.4.33.txt

```diff
@@ -235,3 +225,5638 @@ for series in $SRU_SERIES; do
+ cloud-init status --wait --long
+
+status: done
+time: Fri, 17 Jan 2020 21:26:38 +0000
+detail:
+DataSourceAzure [seed=/dev/sr0]
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@104.208.243.180 -- dpkg-query --show cloud-init
+cloud-init	19.3-41-gc4735dd3-0ubuntu1~16.04.1
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@104.208.243.180 -- sudo cat /run/cloud-init/result.json
+sudo: unable to resolve host SRU-worked-azure
+{
+ "v1": {
+  "datasource": "DataSourceAzure [seed=/dev/sr0]",
+  "errors": []
+ }
+}
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@104.208.243.180 -- grep Trace /var/log/cloud-init.log
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@104.208.243.180 -- systemd-analyze
+Startup finished in 4.550s (kernel) + 22.453s (userspace) = 27.003s
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR ubuntu@104.208.243.180 -- systemd-analyze blame
+          5.838s cloud-init.service
+          3.628s cloud-config.service
+          3.587s cloud-init-local.service
+          2.788s snapd.service
+          2.441s lxd-containers.service
+          2.399s pollinate.service
+          2.031s snapd.seeded.service
+          1.843s dev-sda1.device
+          1.307s console-setup.service
+          1.264s accounts-daemon.service
+           985ms apparmor.service
+           851ms networking.service
+           836ms lvm2-monitor.service
+           780ms iscsid.service
+           742ms mdadm.service
+           668ms grub-common.service
+           644ms cloud-final.service
+           368ms snapd.socket
+           359ms systemd-modules-load.service
+           348ms lxd.socket
+           253ms systemd-tmpfiles-setup-dev.service
+           249ms systemd-timesyncd.service
+           244ms systemd-logind.service
+           220ms rsyslog.service
+           181ms open-iscsi.service
+           155ms systemd-journal-flush.service
+           155ms systemd-udev-trigger.service
+           154ms systemd-journald.service
+           146ms resolvconf.service
+           134ms irqbalance.service
+           118ms keyboard-setup.service
+           109ms apport.service
+            96ms systemd-udevd.service
+            94ms ufw.service
+            85ms ssh.service
+            85ms polkitd.service
+            77ms systemd-remount-fs.service
+            74ms ondemand.service
+            64ms kmod-static-nodes.service
+            64ms dev-mqueue.mount
+            55ms plymouth-read-write.service
+            53ms systemd-random-seed.service
+            52ms systemd-machine-id-commit.service
+            48ms sys-kernel-debug.mount
+            37ms systemd-update-utmp.service
+            33ms systemd-user-sessions.ser
```

Comment:
This looks like the issue?

Reply:
I'm not certain what you were highlighting here as these are 52 milliseconds not minutes.
I think the issue on the bionic system was an intermittent network timeout on ssh-import-id. 

Also I'm seeing Azure in general retry on waiting for availability of their ssh cloud-int certificates during setup. Their retry sleeps for 10 + 1 seconds between attempts.